modules/pmg_qt/pymol_qt_gui.py

```diff
@@ -929,9 +932,9 @@ def command_set_cursor(self, i):
         return self.lineedit.setCursorPosition(i)
 
     def update_progress(self):
-        progress = int(self.cmd.get_progress() * 100)
+        progress = self.cmd.get_progress()
         if progress >= 0:
-            self.progressbar.setValue(progress)
+            self.progressbar.setValue(progress * 100)
```

Comment:
Any reason for this change? Seems to undo the changes made in 11325a6014565700be9673236c9b7b6c856ea6d9

Reply:
Those changes were in the branch qt6, from which I started. For now, I'll make a PR without them, then.